Directory.Build.props

```diff
@@ -2,8 +2,10 @@
   <PropertyGroup>
     <RepositoryName>wpf</RepositoryName>
     <WindowsDesktopARM64Support>true</WindowsDesktopARM64Support>
-    <TargetFramework>net8.0</TargetFramework>
-    <TargetFrameworkVersion>8.0</TargetFrameworkVersion>
+    <TargetFramework>net481</TargetFramework>
+    <TargetFrameworkVersion>4.8.1</TargetFrameworkVersion>
```

Comment:
Super interested if this is even needed? They must use it elsewhere? SDK makes sure this gets set automatically, so the usage of this property must before that logic runs.